Sources/BuyMeACoffee/BMCColor.swift

```diff
@@ -21,27 +36,46 @@ public enum BMCColor: String, CaseIterable {
     
     public static let `default`: Self = .yellow
     
-    internal var background: UIColor? {
-        UIColor(named: rawValue, in: .module, compatibleWith: nil)
+    internal var background: CustomColor? {
+        if #available(iOS 13.0, OSX 10.15, *) {
+            return Color(UIColor(named: rawValue, in: .module, compatibleWith: nil)!)
```

Comment:
Why does it return either a `Color` or `UIColor` depending the platform?

Reply:
Good callout, no need to! Fixed.